cluster/manifests/stackset-controller/01-stack-crd.yaml

```diff
@@ -349,7 +349,7 @@ spec:
                 - maxReplicas
                 - metrics
                 type: object
-{{- if eq .Cluster.ConfigItems.stackset_configmap_support_enabled "true" }}
+{{- if eq .Cluster.ConfigItems.stackset_configuration_rsc_enabled "true" }}
```

Comment:
@katyanna initially you had this depend on either config-map or secret. I think that is a better approach because it's harder to keep in mind two different config-items have to be changed.

I would propose this:

```
{{- if or (eq .Cluster.ConfigItems.stackset_configmap_support_enabled "true") (eq .Cluster.ConfigItems.stackset_secret_support_enabled "true") }}
```

Reply:
I agree but it broke the pipeline. This was a test that seem to have worked but maybe it's related to the indentation, I'll try it again.